index.html

```diff
@@ -0,0 +1,152 @@
+                        event and how it has evolved over time.</p>
+                   </div>
+                   <div class="p-content bg-color d-flex flex-lg-column  flex-md-column gap-1 justify-content-between align-items-center  p-4 ">
+                    <img src="imgs/icons3.png" width="44px" alt="first program icon">
+                    <h6 class="font-color">CC Inigt</h6>
+                    <p class="text-white">The director will shed light on the strategies and efforts that
+                        have contributed to the event's triumphs and the challenges that
+                        have been overcome.</p>
+                   </div>
+                </div>
+                <a href="#" class="no-diplay d-sm-none d-md-block d-lg-block text-center mb-lg-1 text-white p-sm-4">SEE THE WHOLE PROGRAM</a>
+                <div class="p-2 m-auto"><a href="#" class="d-sm-block d-md-none d-lg-none text-center mb-btn">Join TechNO 2015</a></div>
+            </div>
+        </section>
+
+        <!---speaker section-->
+        <section class="container-fluid featured-professors">
+            <h3 class="font-color text-2xl font-medium text-center">Featured Professors</h3>
+            <span class="main-program-hr"></span>
+            <article class="professors-list" id="professors-list">
+            </article>
+            <button type="button" class="more-professors-btn" id="more-professors-btn">More..</button>
+          </section>
+
+          <!-----footer section-->
+          <section class="partners container-fluid ">
+            <h3 class="text-center text-dark p-lg-5 p-md-4 p-sm-2 fw-bolder">Partners</h3>
+            <div class="d-flex justify-content-around flex-wrap gap-2 align-items-center p-2 m-auto ">
+                <img src="imgs/mozilla 1.png"  width="150px" alt="partners logo"/>
+                <img src="imgs/naver 1.png" width="150x" alt="partners logo"/>
+                <img src="imgs/googlekorea 1.png" width="150px" alt="partners logo"/>
+                <img src="imgs/daumkakaologo 1.png" width="150px" alt="partners logo"/>
+                <img src="imgs/airbnb 1.png" width="150px" alt="partners logo"/>
+            </div>
+          </section>
+
+          <!----footer-->
+          <section class="container-fluid bg-white footer p-5 m-auto">
+            <div class="d-flex justify-content-around align-items-center g-2">
+                <img src="imgs/logo.jpg" width="200" alt= "footer logo">
+                <div class="d-flex flex-column gap-3">
+                    <p class="flex-nowrap">2023 Creative Commons Korea. Some Rights Reserved.</p>
+                    <p class="d-lg-block d-md-block no-diplay">달리 정하지 않은 웹사이트의 모든 내용은 크리에이티브 커먼즈 저작자표시 2.0 대한민국 라이선스에 따라 이용할 수 있습니다.
+                        자세한 이용기능 내용과 허락조건은 http://creativecommons.org/licences/by/2.0/kr 를 보세요.</p>
+                </div>
+            </div>
+          </section>
```

Comment:
## Please pay attention to the details since it is very important to have your design identical to the given design so here in the mobile version in home page there is no partner and footer section so kindly remove them from here 🔥💯🤩